remotes/fixupoptions.go

```diff
@@ -114,3 +117,13 @@ func WithAutoBundleUpdate() FixupOption {
 		return nil
 	}
 }
+
+// WithPushImages authorize the fixup command to push missing images in the registry that are
```

Comment:
*authorizes

Also, could the comment be a little more explicit regarding exactly which registry + repo this pushes the image into?
There are two interpretations here - the registry and repo that are referenced in the bundle, or the target registry + repo of the push.